src/TourPortal.js

```diff
@@ -546,6 +547,7 @@ class TourPortal extends Component {
 
             <Close onClick={onRequestClose} />
           </Guide>
+          <Children children={this.props.children} />
```

Comment:
What if we pass directly `{this.props.children}` and it's responsibility to end-user to style that `elem`?

Reply:
If we did that end-users would had to discover `position` and `z-index` values. 
These 2 values are required to show children on top of mask. `position: fixed; z-index: 100000;`

We could just use one of the positioning properties like `top: 0` and remove others. (Or remove them altogether and mention it in docs.)
In this case, `reactour__children` div would have size of its child element so it wouldn't interfere with interaction with the highlighted `elem` and final positioning of child elements' and their container's would be in end-users responsibility.

Reply:
*small update* it requires at least `top` or `bottom` in current method.

Reply:
Thanks @ahmedbera for the updates. 
Since the end-user have to set position attrs to show the `elem` (otherwise remains outside the viewport), I am still thinking that it's pretty similar to let the user set also `position` and `z-index` allowing to set the number needed (maybe have to deal with other `z-index` numbers and so on) and the `position` value that best fit for each situation (maybe `absolute` instead of `fixed`). 
Another thing is that this way, could be rendered the DOM elems needed and when no needed no empty `div` is rendered.
Thanks again.